app/assets/stylesheets/application.scss

```diff
@@ -6,3 +6,4 @@
 @import "bento.scss";
 @import "aleph.scss";
 @import "feedback.scss";
+@import "results.scss";
```

Comment:
I totally agree and usually do it that way but our original compile wasn't letting us use variables from other sheets - let me see if we can use them here.